### PR TITLE
[spoteus][spoteus_demo] add spoteus_demo package

### PR DESCRIPTION
**Pull request by sktometometo**
Add spoteus_demo package

This package includes

- Sample code for demonstration of spoteus
- Sample data for GraphNav

This package download sample data from GoogleDrive with jsk_data. So you don't need to specify an absolute path to sample graphnav file.

Currently, https://github.com/jsk-ros-pkg/jsk_common/pull/1685 is required to download sample data.

### CODE REVIEW

**Comment by sktometometo** [comment]
Confimed that it works well with a robot.

Before to use this, workspace update is necessary.

File: jsk_spot_robot/spoteus_demo/scripts/download_autowalk_data.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python
+
+from jsk_data import download_data
+
+
+def main():
+    PKG='spoteus_demo'
```
Comment (mqcmd196) [inline jsk_spot_robot/spoteus_demo/scripts/download_autowalk_data.py:7]: Could you insert the spaces like `PKG = 'spoteus_demo` ?

File: jsk_spot_robot/spoteus_demo/scripts/download_autowalk_data.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    download_data(
+            pkg_name=PKG,
+            path='autowalk/eng2_73b2_to_81c1_night.walk.tar.gz',
+            url='https://drive.google.com/uc?id=1PXgMsmN3uKPG3YAap9vdfrBduEiLgAl8',
+            md5='a79dd142013b12c0babb4400be583739',
+            extract=True
+            )
+
+
+if __name__=='__main__':
```
Comment (mqcmd196) [inline jsk_spot_robot/spoteus_demo/scripts/download_autowalk_data.py:18]: same as the above.

**Review by mqcmd196 — APPROVED** [review]

File: jsk_spot_robot/spoteus_demo/scripts/download_autowalk_data.py (hunk 1 above)
Comment (sktometometo) [reply jsk_spot_robot/spoteus_demo/scripts/download_autowalk_data.py]: Fixed format

File: jsk_spot_robot/spoteus_demo/scripts/download_autowalk_data.py (hunk 2 above)
Comment (sktometometo) [reply jsk_spot_robot/spoteus_demo/scripts/download_autowalk_data.py]: Fixed format

**Pull request merged**